src/main/resources/de/unijena/cheminf/mortar/message/Message_en_GB.properties

```diff
@@ -544,4 +544,26 @@ IMoleculeFragmenter.CycleFinderOption.relevant.tooltip = Relevant cycles of a gr
 IMoleculeFragmenter.CycleFinderOption.tripletShort.displayName = triplet short
 IMoleculeFragmenter.CycleFinderOption.tripletShort.tooltip =  The shortest cycle through each triple of vertices; allows to generate the envelope rings of some molecules (e.g. naphthalene) without generating all cycles
 IMoleculeFragmenter.CycleFinderOption.vertexShort.displayName = vertex short
-IMoleculeFragmenter.CycleFinderOption.vertexShort.tooltip = The shortest cycles through each vertex; linear independence is not checked; in practice similar to MCB
\ No newline at end of file
+IMoleculeFragmenter.CycleFinderOption.vertexShort.tooltip = The shortest cycles through each vertex; linear independence is not checked; in practice similar to MCB
+##AlkylStructureFragmenter##
+AlkylStructureFragmenter.displayName = Alkyl Structure Fragmenter
+AlkylStructureFragmenter.fragmentSaturationSetting.tooltip = Defines how open valences resulting from bond breakages during fragmentation should be saturated
+AlkylStructureFragmenter.fragmentSaturationSetting.displayName = Fragment saturation setting
+AlkylStructureFragmenter.keepNonFragmentableMoleculesSetting.tooltip = Defines retention of non-fragmentable molecules
```

Comment:
This doesn't tell you anything. Please explain in more detail what the setting does in the tooltip.